### PR TITLE
First pass updating docs structure

### PR DESCRIPTION
**Pull request by dakotabenjamin**

### CODE REVIEW

File: docs/developers/contributing-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Code contributions
+
+Project and code leads are experienced HOT volunteer and staff developers and the main points of contact for the project. They are also the final reviewers of issues and pull requests. Code leads will review pull requests and provide feedback. The purpose of this role is to help contributors,
```
Comment (petya-kangalova) [inline docs/developers/contributing-guidelines.md:3]: To explain here HOT_tech team role and KLL as maintainers. HOT_tech merges to masters, whereas KLL merges to develop and other developers/volunteers can contribute. https://github.com/hotosm/tasking-manager/pull/5116

File: docs/developers/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Prerequisites
```
Comment (petya-kangalova) [inline docs/developers/prerequisites.md:1]: @dakotabenjamin My thinking is that maybe this is not required as a separate section but we make sure any prerequisites are included in the dev setup section above. We can discuss on Tuesday :)

File: docs/developers/project-overview.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Project Overview
+
+This doc will give an overview of the project. Project vision, etc.
```
Comment (petya-kangalova) [inline docs/developers/project-overview.md:3]: @dakotabenjamin thinking that section is not needed as the READ.ME file should give that project overview.

File: docs/developers/contributing-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+4. Apply ESLint and [prettier](https://prettier.io/) style guide rules for the frontend code.
+5. Export translatable strings with `make refresh-translatables` and include them in your commit.
+
+## Code collaboration and version control
+
+*Note: We use **git flow** as our branching model. Read more [here](https://nvie.com/posts/a-successful-git-branching-model/) and [here](https://jeffkreeftmeijer.com/git-flow/), and refer to this [cheatsheet](https://danielkummer.github.io/git-flow-cheatsheet/) if you aren’t familiar with it yet. You probably want to install a [helper application](https://github.com/nvie/gitflow/wiki/Installation) to facilitate the flow a bit.*
+
+* Use branches in the tasking-manager project. This allows others to rebase your branch when they are reviewing or to continue started work. We follow git flow’s naming convention
+  - `feature/ISSUENUMBER-SHORT-TITLE-SEPARATED-BY-HYPHENS` for general new features you are working on
+  - `hotfix/ISSUENUMBER-SHORT-TITLE-SEPARATED-BY-HYPHENS` for important bug fixes that need to go into the main releases as soon as possible
```
Comment (Aadesh-Baral) [inline docs/developers/contributing-guidelines.md:22]: How about adding a new branch `coldfix` or `bugfix` which fixes bugs but isn't that urgent to be deployed to main release as soon as possible.

**Pull request merged**